feat: initial (#1)
fastify/otel: README.md

# @fastify/otel

[![CI](https://github.com/fastify/otel/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/fastify/otel/actions/workflows/ci.yml)

<!-- [![NPM version](https://img.shields.io/npm/v/@fastify/otel.svg?style=flat)](https://www.npmjs.com/package/fastify-plugin) -->

> [![neostandard javascript style](https://img.shields.io/badge/code_style-neostandard-brightgreen?style=flat)](https://github.com/neostandard/neostandard)

Reviewer: In my opinion, there's no need to perpetuate these badge things on new repos. For the most part, they were designed to show CI statuses. We use GHA, so such statuses are directly integrated into the repo already (the branch will have a green or red thing with a direct link to the results).

Author: SGTM, I mostly took this directly from `fastify-plugin` to keep some sort of standard across the `README`; agree with you, I'll remove them away

`fastify-plugin` is a plugin helper for [Fastify](https://github.com/fastify/fastify).

> OpenTelemetry auto-instrumentation library.

Reviewer: I don't understand what this is trying to convey. Why are we talking about `fastify-plugin`?

Author: I think is showing some outdated version, mine says `@fastify/otel`, but let me see if a push fixes it (TL;DR took the `README` from `fatify-plugin` to follow the format)

## Install

```sh
npm i @fastify/otel
```

## Usage

> `@fastify/otel` works as a metric creator as well as instrumentation for you fastify application.

Reviewer: ```suggestion
`@fastify/otel` works as a metric creator as well as application performance monitor for your Fastify application.
```

> Its usage requires to be set before you start defining your routes and other plugins in order to cover up the most of your fatify server.

Reviewer: ```suggestion
It must be configured before defining routes and other plugins in order to cover the most of your Fastify server.
```
